### PR TITLE
Clarify the sources usage on PolicyServer CRs

### PR DESCRIPTION
**Pull request by viccuad**

### CODE REVIEW

File: docs/reference/sources_yaml.md [hunk 1]
```diff
@@ -13,13 +13,16 @@ doc-topic: [operator-manual, sources.yaml]
   <link rel="canonical" href="https://docs.kubewarden.io/reference/sources_yaml"/>
 </head>
 
-## The `sources.yaml` file
+You can tune the push-pull behavior of `kwctl` and `policy-server` binaries
+using the `sources.yaml` file, configured via the `--sources-path` argument which specifies the file.
```
Comment (jhkrug) [inline docs/reference/sources_yaml.md:17]: Very minor suggestion.

```suggestion
You can tune the push-pull behavior of `kwctl` and `policy-server` binaries with the `--sources-path` argument, 
for specifying the path to the `sources.yaml` file.
```

File: docs/reference/sources_yaml.md [hunk 2]
```diff
@@ -13,13 +13,16 @@ doc-topic: [operator-manual, sources.yaml]
...
 
-You can tune the push-pull behavior of `kwctl` and `policy-server` using the `sources.yaml` file.
+To configure a PolicyServer CR, set its `spec.insecureSources` and `spec.sourceAuthorities` fields. The format
+of this fields corresponds with their sections below.
```
Comment (jhkrug) [inline docs/reference/sources_yaml.md:20]: Again, a very minor suggestion.

```suggestion
of these fields corresponds with the respective sections below.
```

**Review by jhkrug — CHANGES_REQUESTED** [review]
Some very minor suggestions.

**Comment by flavio** [comment]
Thanks for the improvements to both of you!

**Pull request merged**